cmd/gateway/commands.go

```diff
@@ -377,3 +384,29 @@ func createSleepCommand() *cobra.Command {
 
 	return cmd
 }
+
+func parseFlagKeysAndValues(flags *pflag.FlagSet) (flagKeys, flagValues []string) {
+	flagKeys = []string{}
+	flagValues = []string{}
```

Comment:
Can we use a map to link the values directly to their keys?

Reply:
I think the issue of this was "ease of conversion to attributes". @pleshakov probably has a better answer but I think a high level was "best we can do currently is slices of basics" or something like that. I'll let him give a better answer 😅

Reply:
here we can use a map just fine.

However, in the telemetry package, it is necessary to use slices.

The exporter library code gen will generate scheme and Attributes() method based on data struct we define.

Because there is no standard conversion between a map and OTel Attributes, that's why we use two slices.

Reply:
No sense in using a map then. I was hoping the final result would be able to use it, but I don't think we need to use it as an intermediary.